redaxo/src/addons/be_style/plugins/redaxo/scss/_form.scss

```diff
@@ -273,3 +273,8 @@ label.required::after {
 .form-control {
     height: auto;
 }
+
+// Overwrite Bootstrap searchfield definition 
+input[type="search"]::-webkit-search-cancel-button {
+    -webkit-appearance: searchfield-cancel-button;
```

Comment:
Bei Non-Webkit Browsern wird der Button angezeigt?

Reply:
siehe screenshot im PR vom visual regression testing.

dort sieht man auch kein search-field

Reply:
Zumindest im Safari wird das X erst angezeigt, wenn es eine Eingabe gibt (glaube ich).
Eventuell ist es im Chrome genauso.

Reply:
Ich frage deshalb nach Screenshots, da Bootstrap es damals nur für -webkit Browser angepasst hat. Ich vermute um ein einheitliches Aussehen aller Browser zu gewährleisten. Da andere Browser aber nicht angepasst wurden, vermute ich dass dieser Cancel-Button damals in anderen Browsern nicht vorhandenw war (oder heute schon) und wir sollten ggf. diesen Button selbst einbauen.

Reply:
Wie auch im Issue erwähnt, taucht es z.B. auch auf dem iPad nicht auf